components/PodLogin.jsx

```diff
@@ -0,0 +1,77 @@
+import { Logo } from './logo';
+import { Formik } from 'formik'
+import { ArrowRight as ArrowRightIcon } from './icons'
+import { Input } from './inputs';
+import { ArrowCircleRight } from './icons'
+import Image from 'next/image';
+import forestImage from '../public/forest-landscape.png';
```

Comment:
This is a different url than the one in tailwind. Should it be?

Reply:
whoop, no, didn't look at this, good catch!